Tests/Integration/Transacting/MobileCoinClientInternalIntTests.swift

```diff
@@ -287,4 +287,173 @@ class MobileCoinClientInternalIntTests: XCTestCase {
             }
         }
     }
+    
+#if swift(>=5.5)
+
+    @available(iOS 15.0, *)
+    func testDynamicAccountCreation() async throws {
+        // try XCTSkip()
+        let minFee = IntegrationTestFixtures.fee
+         let minMOBUSDFee: UInt64 = 2650
+
+        func verifyBalances(
```

Comment:
```suggestion
        func verifyAndReturnBalances(
```

Reply:
Hmm.. what do we need to return the balances for here?  This is just the test verification code - making sure all the new account balances match the expected amount totals...

Reply:
Resolving for now, but lmk if you see any real need for this after review of comment - thanks!